catalogue/webapp/pages/item.js

```diff
@@ -335,17 +335,19 @@ const ItemPage = ({
       <IIIFViewer>
         <IIIFViewerMain>
           <Paginator {...mainPaginatorProps} render={XOfY} />
-          <img
-            className={classNames({
-              'block h-center': true,
-              [spacing({ s: 2 }, { margin: ['bottom'] })]: true,
-            })}
+
+          <ResponsiveImage
             width={largestSize.width}
             height={largestSize.height}
-            src={urlTemplate({
-              size: `max`,
+            url={urlTemplate({ size: `max` })}
+            sizesQueries={`(min-width: 860px) 800px, calc(92.59vw + 22px)`}
```

Comment:
How are we calculating these?
If it's the image lint thing, I wonder how sustainable that is?

Reply:
Could these not be somehow linked to our `supportedSizes`?

Reply:
Because the images can appear at any width at any screen size (unrelated to `supportedSizes`), there isn't a rational way to marry the two up. I agree it doesn't feel great, but I don't know that there's a better alternative.